### ISSUE TITLE
non existing import path in +config

### ISSUE THREAD

**Issue by royalswe** [brillout/vite-plugin-ssr#843]
### Description

accidentally entered a wrong import path in +config.
The +config file is in the top level.
Off course the error went away when corrected the import path.

### Error Message + Error Stack

```shell
error when starting dev server:
Error: [vite-plugin-ssr@0.4.120][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error. (The error's stack trace is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at getCodeFilePath (/Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:333:24)
    at resolveConfigElement (/Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:239:22)
    at /Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:138:35
    at Array.forEach (<anonymous>)
    at /Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:137:63
    at Array.forEach (<anonymous>)
    at loadConfigData (/Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importUserCode/v1-design/getConfigData.js:118:13)
    at async resolveConfig (/Users/royal/www/vite-ssr-ssl/base-url-server-vue-v1/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/config/index.js:28:48)
```

**Comment by brillout** [comment]
Thanks for the bug report. I'll fix it tomorrow.

**Comment by brillout** [comment]
Fix released in `0.4.121`.
